qemu/qemu.go

```diff
@@ -809,6 +819,15 @@ func (vhostuserDev VhostUserDevice) QemuParams(config *Config) []string {
 		devParams = append(devParams, "logical_block_size=4096")
 		devParams = append(devParams, "size=512M")
 		devParams = append(devParams, fmt.Sprintf("chardev=%s", vhostuserDev.CharDevID))
+	case VhostUserFS:
+		driver = VhostUserFS
+		devParams = append(devParams, string(driver))
+		devParams = append(devParams, fmt.Sprintf("chardev=%s", vhostuserDev.CharDevID))
+		devParams = append(devParams, fmt.Sprintf("tag=%s", vhostuserDev.Tag))
+		devParams = append(devParams, fmt.Sprintf("cache-size=%dG", vhostuserDev.CacheSize))
```

Comment:
coveralls seems to be working again, but actually I do have a question.  Is 0 a valid value for vhostuserDev.CacheSize?  I ask as we don't check this value in the Valid method, which led me to wonder whether cache-size=0G was a valid option.